src/client.rs

```diff
@@ -10,9 +10,6 @@ use types::*;
 /// Generalized Electrum client that supports multiple backends. This wraps
 /// [`RawClient`](client/struct.RawClient.html) and provides a more user-friendly
 /// constructor that can choose the right backend based on the url prefix.
-///
-/// **This is only available with the `default` features.**
```

Comment:
I would keep a similar comment that maybe says "This is available only with the default features, or if `proxy` and one ssl implementation are enabled"

Reply:
done